<launch>

<param name="image_topic_name" value="/uav/camera/left/image_rect_color" />
<node name="pencil_filter_node" pkg="gate_fusion" type="pencil_filter_node" output="screen">
<!-- <node name="gate_map_node" pkg="gate_fusion" type="gate_map_publisher" output="screen" launch-prefix="gdb -ex run &#45;&#45;args"> -->
</node>

</launch>